frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_file_exporter_widget.dart

```diff
@@ -55,33 +80,35 @@ class _FileExporterWidgetState extends State<FileExporterWidget> {
           onPressed: () async {
             await getIt<FilePickerService>()
                 .getDirectoryPath()
-                .then((exportPath) {
-              Navigator.of(context).pop();
+                .then((exportPath) async {
+              if (exportPath != null && cubit != null) {
+                final views = cubit!.state.selectedViews;
+                await _AppFlowyFileExporter.exportToPath(exportPath, views);
+                Log.debug(views);
+                _showToast(LocaleKeys.settings_files_exportFileSuccess.tr());
+              } else {
+                _showToast(LocaleKeys.settings_files_exportFileFail.tr());
+              }
+              if (mounted) {
+                Navigator.of(context)
+                  ..pop()
+                  ..pop();
+              }
             });
           },
         ),
       ],
     );
   }
 
-  FutureBuilder<dartz.Either<WorkspaceSettingPB, FlowyError>>
-      _buildFileSelector(BuildContext context) {
-    return FutureBuilder<dartz.Either<WorkspaceSettingPB, FlowyError>>(
-      future: FolderEventReadCurrentWorkspace().send(),
-      builder: (context, snapshot) {
-        if (snapshot.hasData &&
-            snapshot.connectionState == ConnectionState.done) {
-          final workspaces = snapshot.data?.getLeftOrNull<WorkspaceSettingPB>();
-          if (workspaces != null) {
-            final apps = workspaces.workspace.apps.items;
-            return BlocProvider<SettingsFileExporterCubit>(
-              create: (_) => SettingsFileExporterCubit(apps: apps),
-              child: const _ExpandedList(),
-            );
-          }
-        }
-        return const CircularProgressIndicator();
-      },
+  void _showToast(String message) {
+    ScaffoldMessenger.of(context).showSnackBar(
+      SnackBar(
+        content: FlowyText(
+          message,
+          color: Theme.of(context).colorScheme.onSurface,
```

Comment:
`appearance.dart` says that the backgroundColor is `colorScheme.primary`

```dart
      snackBarTheme: SnackBarThemeData(
        backgroundColor: colorScheme.primary,
        contentTextStyle: TextStyle(color: colorScheme.onSurface),
      ),
```
Should we add this in the theme file rather than once ehere?

Reply:
Yes. Agree with you.